### DISCUSSION TITLE
Can't reach database server at `aws-0-ap-south-1.pooler.supabase.com`:`5432`

### DISCUSSION THREAD

**Discussion by viraj-mahida** [supabase/supabase#22473] [Questions]
Sometimes the queries works fine but some times it says: "Can't reach database server at `aws-0-ap-south-1.pooler.supabase.com`:`5432` Please make sure your database server is running at `aws-0-ap-south-1.pooler.supabase.com`:`5432`." 

This issue is only on local development environment, on the published live app there is no error like this. 

The main thing is this error comes once or twice a day, other times it all runs fine. Which means there is not typo or issue with connection string.

I am working on a Next.js web app with Prisma.

**Comment by Manthan-Kuber** [comment]
Having the same issue. Would really appreciate any help !

**Comment by auauwolff** [comment]
I have the same issue why that's happening ?
